tronweb/coffee/views.coffee

```diff
@@ -34,12 +34,23 @@ window.dateFromNow = (string, defaultString='never') ->
 
 
 window.getDuration = (time) ->
-    [time, ms] = time.split('.')
-    [hours, minutes, seconds] = time.split(':')
-    moment.duration
-        hours: parseInt(hours)
-        minutes: parseInt(minutes)
-        seconds: parseInt(seconds)
+    if time.indexOf("day") != -1
```

Comment:
https://en.wikipedia.org/wiki/Don%27t_repeat_yourself  
There are some duplicated codes here. Could you fix that?